ironfish/src/rpc/routes/wallet/burnAsset.ts

```diff
@@ -75,6 +82,7 @@ router.register<typeof BurnAssetRequestSchema, BurnAssetResponse>(
     request.end({
       assetId: burn.assetId.toString('hex'),
       hash: transaction.hash().toString('hex'),
+      name: BufferUtils.toHuman(assetResponse.name),
```

Comment:
We always send this back as hex16 encoded `assetName` field, like in other RPC. You can toHuman on the other side, or people cannot get the raw asset name buffer.

```suggestion
      assetName: assetResponse.name.toString('hex'),
```

Reply:
also updated mintAssetResponse name to `hex`, instead of `utf8`